substrate-node/pallets/pallet-tft-bridge/src/benchmarking.rs

```diff
@@ -75,7 +75,7 @@ benchmarks! {
     }: _(RawOrigin::Signed(caller.clone()), target_stellar_address.clone(), amount)
     verify {
         let burn_id = 1;
-        let tx = TFTBridgeModule::<T>::burn_transactions(burn_id);
+        let tx = TFTBridgeModule::<T>::burn_transactions(burn_id).unwrap();
```

Comment:
Is this safe? a panic i the chain is un-recoverable and the chain can be dead forever

Reply:
Yes its totally safe.
This is not part of the chain runtime, it is benchmark code which used within the benchmark framework to generate estimates about the execution time required for the functions in a pallet.
I can see it used in other TFChain benchmarks and also in some substrate pallet benchmark like vesting pallet.